saleor/app/installation_utils.py

```diff
@@ -270,6 +271,7 @@ def install_app(app_installation: AppInstallation, activate: bool = False):
             mount=extension_data.get("mount"),
             target=extension_data.get("target", AppExtensionTarget.POPUP),
             http_target_method=http_target_method,
```

Comment:
Don't we want to drop `http_target_method`? 🤔  It is marked as deprecated, and you store all data in `settings`.

Reply:
I need this because current resolver still returns it as enum. When I remove deprecated field, I will be able to drop this from DB. 

Same thing with target and mount enums, I still need to keep the old ones, otherwise I risk typed enum in graphql to receive invalid string